src/profile/components/ProfileContainer/ProfileContainer.tsx

```diff
@@ -0,0 +1,30 @@
+import { useState } from "react"
+import { useMedia } from "@/shared/hooks"
+import { mediaQueries } from "@/config"
+import { ProfileDesktop, ProfileMobile } from "@/profile"
+
+export enum OrganizationType {
+  GoogleLLC = "Google LLC",
+  Xiaomi = "Xiaomi",
+  LG = "LG",
+}
+
+export const ProfileContainer = () => {
+  const isSmallAndAbove = useMedia([mediaQueries.sm], [true], false)
+  const [userOrganization, setUserOrganization] = useState("")
```

Comment:
change to `// TODO: fix it after BE changes with available organizations`
  `const [userOrganization, setUserOrganization] = useState<string>("1")`